src/appleseedmaya/exporters/lightexporter.cpp

```diff
@@ -128,17 +128,32 @@ void LightExporter::createEntities(
             asr::ColorEntityFactory::create(colorName.asChar(), params, values));
     }
 
+    auto extraControls = [&](asr::ParamArray& lightParams)
```

Comment:
Do you confirm that `[]` isn't sufficient here?

Reply:
I have to capture the MDependencyNode object *depFnNode*, or pass it as a parameter but the lambda was replaced by a function in https://github.com/luisbarrancos/appleseed-maya2/blob/dcaba8b1297b2c82e63df4cd8c1e4086a6e66a65/src/appleseedmaya/exporters/lightexporter.cpp#L50
perhaps i should use the lambda instead of the function in the anonymous namespace, capture by reference and get rid of the parameter list ?
```
auto extraControls = [&] () ...
```

Reply:
A function is fine.